doc/dev/development.md

```diff
@@ -77,6 +79,13 @@ To run some of Mixer's examples and tests, you need to set up Docker server.
 Please follow [these instructions](https://docs.docker.com/engine/installation/)
 for how to do this for your platform.
 
+### Setting up personal access token
+
+Since Istio requires two-factor authentication, you must setup personal access
+token to enable push via HTTPS. Please follow [these
+instructions](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)
+for how to create a token.
+
```

Comment:
My understanding is that this instruction is for pushing into the istio/mixer instead of the fork.